### ISSUE TITLE
RFE: description of differences between repositories

### ISSUE THREAD

**Issue by whitel** [flatpak/flatpak#3652]
## Linux distribution and version
Fedora Silverblue 32
## Flatpak version
Flatpak 1.6.3
## Description of the problem
If I go to install "gimp" in flatpak I am presented with a choice of where to install it from: fedora or flathub. While I could just choose based on my preferred "provider" in some cases their may be capability differences (e.g. codecs or whatnot). Would be nice if at that prompt there was a further description about why I might want to choose one over the other.

Gimp is purely just an example that made me think of this not nec. representative of something that is different.

**Comment by TingPing** [comment]
That really isn't information flatpak can know about, I'm not sure anything can be done?

**Comment by mwleeds** [comment]
Yeah, conceivably we might be able to show the version available though

**Comment by whitel** [comment]
I would imagine the author of the flatpak may have a "reason" for why they created the new flatpak. Oobviously, they can't be sure they are unique but they may know what capability they are trying to provide the differentiates. I was imagining it would be a "field" in the metadata that the author could optionally fill out. However, maybe it would be so unused that it wouldn't be worth it.

**Comment by TingPing** [comment]
The Fedora repo exists because Fedora wants to package things, there is no meaningful reason to tell the user.

**Comment by whitel** [comment]
> The Fedora repo exists because Fedora wants to package things, there is no meaningful reason to tell the user.

I think that is a pretty broad assumption. I think many things get packaged in flathub because they include libraries that Fedora can't or won't. I also think Fedora may want to package something specifically to not include those libraries..  I don't think this "field" should be required just an option for an author to indicate why a user should choose the one they packaged.

**Comment by TingPing** [comment]
Yes Fedora is bound to more strict legalities but it largely comes down to they simply cannot and will not use Flathub and thus duplicate packaging.
